pandas/_libs/window/aggregations.pyx

```diff
@@ -170,7 +170,7 @@ cdef inline float64_t calc_mean(int64_t minp, Py_ssize_t nobs,
     cdef:
         float64_t result
 
-    if nobs >= minp:
+    if nobs >= minp and nobs > 0:
```

Comment:
Do you suspect that we need to add this wherever `if nobs >= minp` is checked?

Reply:
I'll provide an overview where we have problems with this. Had a quick look at sum, looks like we need it there too